OSX 10.10.2 issues.

Hello,
I have manually installed libtool-2.4.6 and can verify:
`which libtoolize` : `/usr/local/bin/libtoolize`
`libtoolize --version` : `libtoolize (GNU libtool) 2.4.6`
I have manually installed automake-1.9 and can verify:
`which automake` : `/usr/local/bin/automake`
`automake --version` : `automake (GNU automake) 1.9`
I have manually installed autoconf-2.69 and can verify:
`which autoconf` : `/usr/local/bin/autoconf`
`autoconf --version` : `autoconf (GNU Autoconf) 2.69`

Error 1:
since autogen.sh uses glibtoolize, I:
`sudo ln -s /usr/local/bin/libtoolize glibtoolize`
and can verify:
`which glibtoolize` : `/usr/local/bin/glibtoolize`
Addressing Issue #1200 
Now, ./autogen.sh properly executes. see [gumbo-parser.log](http://pastebin.com/L3zVYWv2)

Error 2:
`./configure --prefix=/usr/local/` yields:
`configure: creating ./config.status`
`config.status: error: cannot find input file: Makefile.in`
So I run:
`aclocal`
`autoconf`
`autoheader` : `error: AC_CONFIG_HEADERS not found in configure.ac`
so I uncomment `#AC_CONFIG_HEADERS` from configure.ac and rerun:
`autoheader`
`automake --add-missing`: `Makefile.am: C objects in subdir but AM_PROG_CC_C_O not in configure.ac`
So I add `AM_PROG_CC_C_O` to configure.ac under `# Checks for programs.` and rerun
`aclocal`
`autoconf`
`autoheader`
`automake --add-missing`
`./configure --prefix=/usr/local/`

Error 3:
`make` : `/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-am`
`make[1]: ~/install-sh: No such file or directory`
`make[1]: *** [src/.dirstamp] Error 1`
`make: *** [all] Error 2`
Why it's checking my home directory for install-sh? no idea.
So I
`cp -f install-sh ~/`
`make`
`sudo make install`
`libgumbo.dylib` is now present in `/usr/local/lib/`
More details in the [log file](http://pastebin.com/L3zVYWv2)

--- a/autogen.sh
+++ b/autogen.sh
@@ -15,7 +15,14 @@
 # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 if test -z "$LIBTOOLIZE" -a "`uname`" = "Darwin"; then
-  LIBTOOLIZE=glibtoolize
+	if command -v "glibtoolize" >/dev/null; then
+		LIBTOOLIZE=glibtoolize
+	elif command -v "libtoolize" >/dev/null; then
+		LIBTOOLIZE=libtoolize
+	else
+		echo "autogen.sh: line $LINENO: command glibtoolize or libtoolize not found"
+		exit 1
+	fi
 fi
 
 set -ex

